### DISCUSSION TITLE
Do you have any design tools?

### DISCUSSION THREAD

**Discussion by honghaier-game** [TomSchimansky/CustomTkinter#185] [General]
Hello, I'm very happy to see this project. I'm the developer of another tool software Tkinterdesigner'. I want to integrate the use effect of this project interface into it. Is it possible?
My project address: https://github.com/honghaier-game/TKinterDesigner

**Comment by felipetesc** [comment]
Since Ctk and Tk are compatible, I belive it's possible. If it's going to be easy ? That's something else.

**Reply by felipetesc** [reply]
> Since Ctk and Tk are compatible, I belive it's possible. If it's going to be easy ? That's something else.

PS: It will be nice to have a tool like that
